server/util/bazel/defs.bzl

```diff
@@ -54,7 +90,16 @@ def bazel_pkg_tar(name, versions = [], **kwargs):
         extract_bazel_installation(
             name = "bazel-{}_extract_installation".format(version),
             bazel = ":bazel-{}_crossplatform".format(version),
-            out_dir = "bazel-{}_install".format(version),
+            out_dir = "bazel-{}_outdir".format(version),
+            # Pre-warm repository cache only for bazel 8+, where we've started
+            # using repository_cache combined with MODULE.bazel.lock to make
+            # bazel work without network access.
+            warm_repository_cache = int(version.split(".")[0]) >= 8,
+            # If we are warming the repository cache then we need network access
+            # so that this rule can download the dependencies. Targets that use
+            # this rule can then use the cached dependencies without needing the
+            # network.
+            exec_properties = {"dockerNetwork": "bridge"},
```

Comment:
The comment suggests that we might want to set this conditionally on the value of `warm_repository_cache`.

Reply:
Done